org.adempiere.base/src/org/adempiere/model/GenericPO.java

```diff
@@ -81,6 +81,16 @@ public GenericPO(String tableName, Properties ctx, ResultSet rs, String trxName)
 		super(new PropertiesWrapper(ctx, tableName), 0, trxName, rs);
 	}
 
+	/**
+	 * @param tableName
+	 * @param ctx
+	 * @param uuID 
+	 * @param trxName
+	 */
+	public GenericPO(String tableName, Properties ctx, String uuID, String trxName) {
+		super(new PropertiesWrapper(ctx, tableName), 0, trxName, null, uuID);
+	}
+

```

Comment:
this constructor should be removed as well

Reply:
Done